static/app/components/events/interfaces/crashContent/exception/utils.spec.tsx

```diff
@@ -0,0 +1,174 @@
+import {urlEncode} from '@sentry/utils';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {renderLinksInText} from 'sentry/components/events/interfaces/crashContent/exception/utils';
+
+describe('Linkify()', function () {
```

Comment:
Looks like these tests will need to change now that we aren't just rendering a link

Reply:
Yep... fixing them now 😞